### PR TITLE
Add nethserver-mail-getmail migration

### PR DESCRIPTION
**Pull request by stephdl**
Provide the migration of nethserver-mail-getmail, we need to look after an action file because getmail is not a service but a cron based task

Refs https://github.com/NethServer/dev/issues/6776

image
image
image
image
image
image
image
image
image
image
image

once installed think to change 

```
[root@NS3 ~]# cat /usr/share/nethesis/nethserver-ns8-migration/apps/nethserver-mail-getmail/bind.env 
# This file is included by ns8-bind-app
MODULE_VOLUMES=""
MODULE_IMAGE_URL="ghcr.io/nethserver/imapsync:latest"
```

### CODE REVIEW

File: api/migration/read [hunk 1]
```diff
@@ -162,6 +164,16 @@ def get_webmail_info():
         return None
 
 
+def get_connector_info():
+    if os.path.isfile('/etc/e-smith/events/actions/nethserver-getmail-conf'):
+        return {
+            "id": "nethserver-mail-getmail",
+            "name": "Connector"
```
[the remark this answers is not in the document]
Comment (andre8244) [reply api/migration/read:171]: We should always use the same name for this application on the UI. Let's choose between "POP3 Connector" and "GetMail" and use that name everywhere (we have used "GetMail" inside the string: "If Webmail, WebTop and/or GetMail are currently installed, they will be migrated too.")

[the remark this answers is not in the document]
Comment (andre8244) [reply api/migration/read:171]: We should always use the same name for this application on the UI. Let's choose between "POP3 Connector" and "GetMail" and use that name everywhere (we have used "GetMail" inside the string: "If Webmail, WebTop and/or GetMail are currently installed, they will be migrated too.")

**Review by andre8244 — COMMENTED** [review]
Let's always use the same name for the app on the UI https://github.com/NethServer/nethserver-ns8-migration/pull/46/files#r1402205395

**Review by andre8244 — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -159,16 +159,16 @@ Finish command example ::
...
   MIGRATE_ACTION=finish MATTERMOST_VHOST=mattermost.example.com ./migrate
 
-Email, Webtop, Roundcube migration
-----------------------------------
+Email, Webtop, Roundcube migration, Getmail
+-------------------------------------------
 
-As both Webtop and Roundcube depends on the Email application, the
-migration of the three modules must occur at the same time and is
+As both Getmail, Webtop and Roundcube depends on the Email application, the
```
[the remark this answers is not in the document]
Comment (stephdl) [reply README.rst]: done

**Pull request merged**